src/mlpack/methods/pca/pca_impl.hpp

```diff
@@ -33,10 +35,10 @@ PCA<DecompositionPolicy>::PCA(const bool scaleData,
  * @param coeff - PCA Loadings/Coeffs/EigenVectors
  */
 template<typename DecompositionPolicy>
-void PCA<DecompositionPolicy>::Apply(const arma::mat& data,
-                arma::mat& transformedData,
-                arma::vec& eigVal,
-                arma::mat& coeff)
+void PCAType<DecompositionPolicy>::Apply(const arma::mat& data,
+                                         arma::mat& transformedData,
+                                         arma::vec& eigVal,
+                                         arma::mat& coeff)
```

Comment:
This parameter should call "eigvec" or "coeff", they are different in pca.hpp and pca_impl.hpp